src/main/java/com/google/maps/PlacesApi.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2015 Google Inc. All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.google.maps;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.maps.errors.ApiException;
+import com.google.maps.internal.ApiConfig;
+import com.google.maps.internal.ApiResponse;
+import com.google.maps.model.LatLng;
+import com.google.maps.model.PlaceDetails;
+
+import java.util.List;
+
+/**
+ * The Google Places API enables you to get data from the same database used by Google Maps and
+ * Google+ Local. Places features more than 100 million businesses and points of interest that are
+ * updated frequently through owner-verified listings and user-moderated contributions.
+ * <p/>
+ * <p>See also: <a href="https://developers.google.com/places/web-service/">Places API Web Service
+ * documentation</a>.
+ */
+public class PlacesApi {
+
+  private PlacesApi() {
+  }
+
+  /*
+   * Please note: We are specifically not implementing:
+   *  - Nearby Search
+   *  - Radar Search
+   *  - Place Add
+   *  - Place Autocomplete
+   */
+
+  /**
+   * The Google Places API Text Search Service is a web service that returns information about a
```

Comment:
First sentence of javadoc should be a very short noun or verb phrase, not a full sentence. e.g. "Performs a text search for places. The Google Places API...". This is pretty important for the API call entry methods.